docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,189 @@
+    'react/button-story-auto-docs.ts.mdx',
+    'vue/button-story-auto-docs.js.mdx',
+    'vue/button-story-auto-docs.ts.mdx',
+    'angular/button-story-auto-docs.ts.mdx',
+    'svelte/button-story-auto-docs.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-## Subcomponents parameter
+<video autoPlay muted playsInline loop>
+  <source
+    src="storybook-button-auto-docs-optimized.mp4"
+    type="video/mp4"
+  />
+</video>
 
-Sometimes it's helpful to document multiple components together. For example, a component library’s ButtonGroup and Button components might not make sense without one another.
+Once the story loads, Storybook infers the relevant metadata (e.g., [`args`](../writing-stories/args.md), [`argTypes`](../api/argtypes.md), [`parameters`](../writing-stories/parameters.md)) and automatically generates a documentation page with this information positioned at the root-level of your component tree in the sidebar.
 
-DocsPage has the concept of a "primary" component defined by the `component` parameter. It also accepts one or more `subcomponents`.
+### Configure
+
+By default, Storybook offers zero-config support for documentation and automatically sets up a documentation page for each story enabled via the `tags` configuration property. However, you can extend your Storybook configuration file (i.e., `.storybook/main.js|ts|cjs`) and provide additional options to control how documentation gets created. Listed below are the available options and examples of how to use them.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-group-story-subcomponents.js.mdx',
-    'common/button-group-story-subcomponents.ts.mdx'
+    'common/storybook-auto-docs-full-config.js.mdx',
+    'common/storybook-auto-docs-full-config.ts.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-![Subcomponents in Docs Page](./docspage-subcomponents.png)
-
-Subcomponent `ArgsTables` will show up in a tabbed interface along with the primary component. The tab titles will correspond to the keys of the subcomponents object.
-
-If you want to organize your documentation differently for component groups, we recommend using MDX. It gives you complete control over how it's displayed and supports any configuration.
-
-## Replacing DocsPage
-
-Replace the DocsPage template with your own to customize its contents.
+| Option        | Description                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------ |
+| `enabled`     | Toggles support for auto-generated documentation pages <br/> `docs: { enabled:false }`                 |
+| `docsPage`    | Disables auto-generated documentation pages created via `tags` <br/> `docs: { docsPage: false }`       |
+| `automatic`   | Enables auto-generated documentation pages for every component <br/> `docs: { docsPage: 'automatic' }` |
```

Comment:
I don't understand the difference between `docs.enabled = false` and `docs.docsPage = false`.

Reply:
Chatted about this and now understand the distinction, so:

```suggestion
| `enabled`     | Toggles support for all documentation pages <br/> `docs: { enabled:false }`                 |
| `docsPage`    | Disables auto-generated documentation pages created via `tags` <br/> `docs: { docsPage: false }`       |
| `automatic`   | Enables auto-generated documentation pages for every component <br/> `docs: { docsPage: 'automatic' }` |
```

I'm still not confident we need both options, but 🤷‍♂️.